### PR TITLE
Add mixin for exporter metrics

### PR DESCRIPTION
**Pull request by BinaryFissionGames**
This PR includes 2 dashboards and 6 alerts for the Snowflake exporter.

### CODE REVIEW

File: mixin/config.libsonnet [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  _config+:: {
+    dashboardTags: ['snowflake-mixin'],
+    dashboardPeriod: 'now-8h',
+    dashboardTimezone: 'default',
+    dashboardRefresh: '30m',
+
+    // for alerts
+    alertsWarningLoginFailures: '30',  // %
+    alertsComputeCreditUsageLimit: '5',  // credits/hr
+    alertsServiceCreditUsageLimit: '1',  // credits/hr
```
Comment (Dasomeone) [inline mixin/config.libsonnet:11]: I wonder how common these credit usage limits would be for orgs of varying sizes 🤔

In the future, this is definitely something we want to be configurable within the integration setup/settings

**Comment by Dasomeone** [comment]
General comment on both dashboards, because I didn't notice this as much in the initial screenshots:

A lot of the panels feel oversized, both in terms of height and width, especially the 24 column wide panels. I'd suggest compressing them to at most 12 columns, which would let you fit most of the dashboard in a single view, without too much scrolling. 
One of those things that looks fine on smaller screens and in cropped screenshots, but the moment you blow it up on a big screen it really shows the difference

**Review by Dasomeone — COMMENTED** [review]
Looks good for now. I generally like the layout, but as mentioned in my other comment I think the panels are, in general, oversized. 

We can iterate on it, but it's a good start! :)

**Review by Dasomeone — APPROVED** [review]

File: mixin/dashboards/snowflake-overview.libsonnet [hunk 2]
```diff
@@ -0,0 +1,1214 @@
...
+    prometheus.target(
+      'snowflake_used_cloud_services_credits{job=~"$job", instance=~"$instance"}',
+      datasource=promDatasource,
+      format='table',
+      legendFormat='__auto'
+    ),
+  ],
+  type: 'table',
+  title: 'Top 5 service cloud service credits used',
+  description: 'The top 5 Services that have the highest compute credit usages.',
```
Comment (v-zhuravlev) [inline mixin/dashboards/snowflake-overview.libsonnet:558]:
```suggestion
  description: 'The top 5 services that have the highest compute credit usages.',
```

**Review by v-zhuravlev — APPROVED** [review]

**Pull request merged**